Training/2.0 Libraries.md

```diff
@@ -52,14 +67,15 @@ Normally we would want to be able to extract the information from tables directl
 [Training2_example.csv](https://github.com/alepgr/gubmes-hda/blob/main/Training/Training2_example.csv)
 
 ```
-
+f = pd.read_csv('Resources/Training2_example.csv')
 ```
 
 ### Task 4a)
 Using your DataFrame from Task 4a), calculate the average systolic blood pressure of Dementia patients.
 
 ```
-
+filtered = f.groupby('Condition')['Systolic Blood Pressure'].mean()
+average = filtered.loc['Dementia']
 ```
```

Comment:
Correct! .groupby() method is an interesting approach, very useful if you had to calculate an average for each patient group.